prisma/seed.js

```diff
@@ -282,6 +284,69 @@ async function dataSeed() {
     adjudicationUpserts.push(adjudicationUpsert);
     console.log(adjudicationUpsert);
   }
+
+  const awards = [
+    {
+      id: 1,
+      title: 'Award 1',
+      is_category: false,
+    },
+    {
+      id: 2,
+      title: 'Award 2',
+      is_category: false,
+    },
+    {
+      id: 3,
+      title: 'Award 3',
+      is_category: true,
+    },
+    {
+      id: 4,
+      title: 'Award 4',
+      is_category: true,
+    },
+  ];
+
+  const awardUpserts = [];
+  for (const award of awards) {
+    const awardUpsert = await prisma.award.upsert({
+      where: {
+        id: award['id'],
+      },
+      update: {
+        title: award['title'],
+        is_category: award['is_category'],
+      },
+      create: {
+        id: award['id'],
+        title: award['title'],
+        is_category: award['is_category'],
```

Comment:
Not a big deal but we can just use `award.id`, `award.title`, and `award.is_category` here.